### PR TITLE
BaSD-MS-17: Checkbox shared component

### PR DESCRIPTION
**Pull request by Pablo-Balbo**
## Description:
Creation of a new Checkbox component. 
It is implemented in the forms corresponding to Postulants, Psychologists and Positions.
### Steps to test Postulant's Checkbox:
1. Run `npm run start` on both front and back-end apps.
2. Search for **Postulants** option on nav bar and click on it.
3. Click **add** or **edit** button to open form.
4. Check if "Available" Checkbox feature are working properly.
### Steps to test Psychologists's Checkbox:
1. Repeat step one.
2. Search for **Psychologists** option on nav bar and click on it.
3. Click **add** or **update** button to open form.
4. Check if "Available" Checkbox feature are working properly.
### Steps to test Positions's Checkbox:
1. Repeat step one.
2. Search for **Positions** option on nav bar and click on it.
3. Click **add** or **update** button to open form.
4. Check if "Available" Checkbox feature are working properly.
### Reviewers:
- @estefrare
- @nicolasblasco 
- @GuidoGlielmi 
- @ValentinaCollareta

### CODE REVIEW

**Comment by nicolasblasco** [comment]
@Pablo-Balbo when we want to update a session, the checkbox that a true are not checked in the UI

File: src/Components/Psychologists/Form/index.js [hunk 1]
```diff
@@ -360,13 +361,19 @@ const Form = () => {
         <h2 className={styles.container}>Availability</h2>
         <div>
           <p>Monday</p>
-          <input
+          <Checkbox
+            label="monday-availability"
+            defaultChecked={mondayBool}
```
Comment (nicolasblasco) [inline src/Components/Psychologists/Form/index.js:366]: @Pablo-Balbo this prop should be "value"

File: src/Components/Psychologists/Form/index.js [hunk 2]
```diff
@@ -360,13 +361,19 @@ const Form = () => {
...
+            onChange={onChangeMondayBool}
+            day="monday"
+          />
+          {/* <input
             label="monday-availability"
             day="monday"
             type="checkbox"
             defaultChecked={mondayBool}
```
Comment (nicolasblasco) [inline src/Components/Psychologists/Form/index.js:374]: @Pablo-Balbo this prop should be "value"

File: src/Components/Psychologists/Form/index.js [hunk 3]
```diff
@@ -360,13 +361,19 @@ const Form = () => {
...
             onChange={onChangeMondayBool}
```
Comment (nicolasblasco) [inline src/Components/Psychologists/Form/index.js:375]: @Pablo-Balbo we should also remove all commented code

**Review by nicolasblasco — APPROVED** [review]

**Review by ValentinaCollareta — APPROVED** [review]

**Review by GuidoGlielmi — APPROVED** [review]

**Pull request merged**